operators/apicurio-registry/subscription.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  labels:
+    operators.coreos.com/service-registry-operator.openshift-operators: ""
+  name: service-registry-operator
+  namespace: apicurio-registry
+spec:
+  channel: 2.x
+  installPlanApproval: Automatic
+  name: service-registry-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: service-registry-operator.v2.6.3
```

Comment:
I think this is a question for @davgordo, but including a startingCSV is usually unnecessary.

Why not accept the default startingCSV of the channel on operator subscriptions?

Reply:
Hopefully, it doesn't apply in this case, but we recently ran into a couple of operators that had problematic CSVs in the upgrade path between the default starting csv and the target. So basically, they got stuck upgrading.

Adding a starting csv that is as recent as possible means that the operator doesn't need to upgrade much on installation. Not strictly necessary (unless we have OLM manifest bugs), but it can streamline the install time for the platform.